examples/common/scripts/vtadmin-up.sh

```diff
@@ -48,9 +50,12 @@ vtadmin-api is running!
   - PID: ${vtadmin_api_pid}
 "
 
+echo "Building vtadmin-web..."
+source "${web_dir}/build.sh"
+
 # Wait for vtadmin to successfully discover the cluster
 expected_cluster_result="{\"result\":{\"clusters\":[{\"id\":\"${cluster_name}\",\"name\":\"${cluster_name}\"}]},\"ok\":true}"
-for _ in {0..300}; do
+for _ in {0..100}; do
```

Comment:
FYI - this was taking a long time when I ran it locally. Shortening to 100 was much faster, and I think 100 is a fine threshold. If it's not found by then, it probably wasn't set up correctly. We can also tune this if it becomes an issue!